.azure-pipelines/yamlFileValidator.yaml

```diff
@@ -7,5 +7,5 @@ jobs:
     displayName: 'npm install'
     inputs:
       verbose: false
-  - script: 'npm run tsc && node .script/yamlFileValidator.js'
+  - script: 'npm run tsc && node .script/TypeScript/yamlFileValidator.js'
```

Comment:
Directory is called `TypeScript` but the code inside is `JavaScript`

Reply:
It is becuase the `npm run tsc ` compile the file to JS, when azure devops run. 
Anyway, I removed this hierarchy. 
I think that we could add language hierarchy In case we will see that it is required to organize the code